lib/evaluation/evaluationClient.ts

```diff
@@ -28,8 +28,10 @@ export class EvaluationClient {
 
   private _results: HandlerCheckResult[];
   private _verifiablePresentation: VerifiablePresentation;
+  private _did: string;
```

Comment:
instead of did we can go with something more explicit like `walletDid` or even `holderDid`

Reply:
Fixed!